package.json

```diff
@@ -3,9 +3,6 @@
   "version": "1.0.0",
   "description": "",
   "main": "bootstrap.min.js",
-  "dependencies": {
-    "sergey": "0.0.13"
```

Comment:
isn't sergey required to build the website (netlify) ??  

real question: i dont know :D

Reply:
Well it did build the Deploy Preview with only the "devDependencies", so that is apparently sufficient. Since you already had sergey, I'll forego any package changes.